tests/unit/records/test_records_receivers.py

```diff
@@ -1066,3 +1067,44 @@ def test_populate_author_count_does_nothing_if_record_is_not_literature():
     populate_author_count(None, record)
 
     assert 'author_count' not in record
+
+
+def test_populate_populate_authors_full_name_normalized():
+    schema = load_schema('hep')
+    subschema = schema['properties']['authors']
+
+    record = {
+        '$schema': 'http://localhost:5000/records/schemas/hep.json',
+        'authors': [
+            {
+                'full_name': 'Vurës, John',
+            },
+            {
+                'full_name': 'Vures, John',
+            },
+        ],
+    }
+    assert validate(record['authors'], subschema) is None
+
+    populate_authors_full_name_normalized(None, record)
+
+    assert record['authors'][0]['full_name_normalized'] == u'vur\xebs, john'
+    assert record['authors'][1]['full_name_normalized'] == u'vures, john'
```

Comment:
Because we are testing receivers in isolation from each other, you can write more in your assertions:
```python
expected = [
    {
        'full_name': 'foo',
        'full_name_normalized': 'foo',
    },
]
result = record['authors']

assert expected == result
```